docs/en/compute/Spark.md

```diff
@@ -167,19 +176,21 @@ Note that in client mode you need to set `--driver-java-options "-Dalluxio.user.
 ### Access Data from Alluxio in HA Mode
 
 If Spark is set up by the instructions in [Configure Spark to find Alluxio cluster in HA mode](#configure-spark-to-find-alluxio-cluster-in-ha-mode),
-you can write URIs using the "`alluxio://`" scheme without specifying cluster information in the authority.
-This is because in HA mode, the address of leader Alluxio master will be served by the internal leader election
-or by the configured Zookeeper service.
+you can write URIs using the "`alluxio://`" scheme without specifying cluster
```

Comment:
NIT: should we get rid of the quotes, since it is already using the backticks?

Reply:
done